CMakeLists.txt

```diff
@@ -14,6 +14,13 @@ endif()
 # Setup modules path
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
 
+# Hunter Cross-platform package manager
+include("cmake/HunterGate.cmake")
+HunterGate(
```

Comment:
is hunter being a problem in windows? this could be the best way to get 3rdparty libs compiled statically on windows too.